component/src/test/java/org/wso2/extension/siddhi/execution/geo/GeocoordinateStreamFunctionProcessorTest.java

```diff
@@ -0,0 +1,157 @@
+ *  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ *  KIND, either express or implied.  See the License for the
+ *  specific language governing permissions and limitations
+ *  under the License.
+ *
+ */
+package org.wso2.extension.siddhi.execution.geo;
+
+import org.apache.log4j.Logger;
+import org.testng.AssertJUnit;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+import org.wso2.siddhi.core.SiddhiAppRuntime;
+import org.wso2.siddhi.core.SiddhiManager;
+import org.wso2.siddhi.core.event.Event;
+import org.wso2.siddhi.core.exception.SiddhiAppCreationException;
+import org.wso2.siddhi.core.query.output.callback.QueryCallback;
+import org.wso2.siddhi.core.stream.input.InputHandler;
+import org.wso2.siddhi.core.util.SiddhiTestHelper;
+import org.wso2.siddhi.core.util.config.InMemoryConfigManager;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.concurrent.atomic.AtomicInteger;
+
+/**
+ * Test case for Geocoordinate Stream Function.
+ */
+public class GeocoordinateStreamFunctionProcessorTest {
+    private static final Logger logger = Logger.getLogger(GeocoordinateStreamFunctionProcessorTest.class);
+    private AtomicInteger count = new AtomicInteger(0);
+    private volatile boolean eventArrived;
+    private static int eventCount = 0;
+
+    @BeforeMethod
+    public void init() {
+        count.set(0);
+        eventArrived = false;
+    }
+
+    /**
+     * Creating test for Geocoordinate Stream Function.
+     *
+     * @throws Exception Interrupted exception
+     */
+    @Test
+    public void geocoordinateStreamFunctionProcessorTest1() throws Exception {
+        logger.info("Test Geocoordinate 1 - Test geocoordinates for given IP");
+        Map<String, String> masterConfigs = new HashMap<>();
+        //The key value is a dummy value. You have to get a key value from ipInfoDB
+        //(IpInfoDB : https://www.ipinfodb.com/ )
+        masterConfigs.put("geo.geocoordinate.apiurl", "http://api.ipinfodb.com/v3/ip-city/?key=" +
+                "39a2202599d94b2432ff1a075f1a35cfe99cb40982d1181adb2c41b6c947a251&ip=");
+        SiddhiManager siddhiManager = new SiddhiManager();
+        long start = System.currentTimeMillis();
+        InMemoryConfigManager inMemoryConfigManager = new InMemoryConfigManager(masterConfigs, null);
+        inMemoryConfigManager.generateConfigReader("geo", "geocoordinate");
+        siddhiManager.setConfigManager(inMemoryConfigManager);
+        String inStreamDefinition = "define stream inputStream (ip String);";
+        String query = ("@info(name = 'query') "
+                + "from inputStream#geo:geocoordinate(ip) "
+                + "select longitude, latitude "
+                + "insert into outputStream;"
+        );
+        SiddhiAppRuntime siddhiAppRuntime = siddhiManager
+                .createSiddhiAppRuntime(inStreamDefinition + query);
```

Comment:
format the two lines

Reply:
Fixed https://github.com/wso2-extensions/siddhi-execution-geo/pull/61/commits/a08d5e724a57f053d7d18ae441ace09bafeb208a